ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/management_api_gnmi.schema.yml

```diff
@@ -39,11 +39,70 @@ keys:
                 ip_access_group:
                   type: str
                   description: ACL name
+          grpc_tunnel:
+            type: list
+            items:
+              type: dict
+              keys:
+                name:
+                  type: str
+                  description: Transport name
+                shutdown:
+                  type: bool
+                  description: Operational status of the gRPC tunnel
+                tunnel_ssl_profile:
+                  type: str
+                  description: Tunnel SSL profile name
+                gnmi_ssl_profile:
+                  type: str
+                  description: gNMI SSL profile name
+                vrf:
+                  type: str
+                  description: VRF name is optional
+                destination:
+                  type: dict
+                  keys:
+                    address:
+                      type: str
+                      description: IP address or hostname
+                    port:
+                      type: int
+                      min: 1
+                      max: 65535
+                      convert_types:
+                      - str
+                      description: TCP Port
+                local_interface:
+                  type: dict
+                  keys:
+                    interface:
```

Comment:
let's use `name` for this key to be consistent with other schemas (snmp_servers and router_bfd local_interface definitions)